spring-grpc-core/src/test/java/org/springframework/grpc/server/DefaultGrpcServerFactoryTests.java

```diff
@@ -13,14 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.springframework.grpc.autoconfigure.server;
```

Comment:
I have no clue why Github insists this is a move of the deleted GrpcServiceDiscover - but it is not. Please disregard. It is a newly added test placeholder